cmd/symbol-check/check_symbol.go

```diff
@@ -0,0 +1,348 @@
+/*symbol-check
```

Comment:
Feel free to create an own repo for these cmd scripts I guess they aren't only useful for gotify.

Reply:
I might need to some time to consider this as `gotify` is really one of the really few go projects that decided to try on go plugin functionality. I think `gomod-cap` is somewhat more like a topic of a dedicated project while `symbol-check` is more like a CI script. I will make a decision on this prior to the final review:) Thanks

Reply:
I'll open up a new [repo](https://github.com/eternal-flame-AD/go-exports) and update current references for symbol check. As for the go mod thing I think that dependency overlay should somehow be built-in in the go Module system. Maybe when the module system was built they did not consider version constraints from the host package of plugins.
I will investigate into this and maybe open a proposal on golang.

Reply:
I have opened a proposal at https://github.com/golang/go/issues/29717.

Let's leave the go-mod util here for now and hear from the official go team for future plans:)